packages/rust/ghaf-kill-switch-app/src/main.rs

```diff
@@ -244,11 +246,20 @@ impl Application for KillSwitch {
                     get_popup(popup_settings)
                 }
             }
+            Message::RefreshStatus => {
+                self.config = Self::get_initial_config();
```

Comment:
Since `get_initial_config()` runs an external command, I think it would be better to wrap it in a `Task` (like `run_killswitch_command()`s above) here.